apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -131,6 +147,13 @@ window.PLOrderBlocks = function (uuid, options) {
       placePairingIndicators();
       setAnswer();
     },
+    start: function () {
+      // the start event is only called once when user starts dragging
+      // this is suitable for drawing the indent grid lines, which is static
+      if (enableIndentation) {
+        drawIndentLocationLines(dropzoneElementId);
+      }
+    },
```

Comment:
I really think this should be in the `create` function. The lines should be created only once at the begining, and then left there. 

I don't like having them appear and disappear for at least a few reasons:
- its is distracting for students to have the lines flashing on and off while working on the problem
- you may want to look at the lines even when you aren't in the middle of dragging a block

Reply:
To clarify, do you want the indent lines to appear when the page finishes loading, or when the student first hovers a block over the answer dropzone? Putting `drawIndentLocationLines` in `create` means the lines will be immediately visible after the page loads even before the student drags anything.

Reply:
yes that's right, they should always be visible before they even touch it. I went ahead and made the change myself so I could merge it. Thanks!